packages/react-devtools-shared/src/__tests__/TimelineProfiler-test.js

```diff
@@ -26,11 +31,16 @@ const ReactVersionTestingAgainst = process.env.REACT_VERSION || ReactVersion;
 const enableSiblingPrerendering =
   false && semver.gte(ReactVersionTestingAgainst, '19.0.0');
 
+// This flag is on experimental which disables timeline profiler.
+const enableComponentPerformanceTrack =
+  React.version.startsWith('19') && React.version.includes('experimental');
```

Comment:
This is the technique I used in `console-test` already for testing enableOwnerStacks only in the experimental branch.

@acdlite You might be able to use this same technique for enableSiblingPrerendering above.